modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -91,10 +91,9 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
                     { $"hb|heartbeatinterval=|{StandaloneCliConfigKeys.HeartbeatIntervalDefault}=", "The publisher is using this as default value in seconds " +
                         "for the heartbeat interval setting of nodes without a heartbeat interval setting.",
                         (int i) => this[StandaloneCliConfigKeys.HeartbeatIntervalDefault] = TimeSpan.FromSeconds(i).ToString() },
-                    // ToDo: Bring back once SkipFirst mechanism is implemented.
-                    //{ "sf|skipfirstevent=", "The publisher is using this as default value for the skip first " +
-                    //    "event setting of nodes without a skip first event setting.",
-                    //    (bool b) => this[StandaloneCliConfigKeys.SkipFirstDefault] = b.ToString() },
+                    { $"sf|skipfirstevent=|{StandaloneCliConfigKeys.SkipFirstDefault}=", "The publisher is using this as default value for the skip first " +
```

Comment:
I find `skipfirstevent` a bit confusing now that we have support for OPC UA events. I know this is 2.5 backwards compatibility material, but maybe we could break it for the sake of clarity.

Reply:
Hehehe, yes, agreed, is it possible to hide it from help?  Will investigate...